src/app/ad-authoring/ad-authoring.state.ts

```diff
@@ -9,15 +9,177 @@ export interface AdAuthoringWorkflowState {
+      ? '<\\/head><body><amp-video layout=\\"fill\\" height=\\"1920\\" width=\\"1080\\" autoplay loop> <source src=\\"' +
+        base64AssetStr +
+        '\\" type=\\"' +
+        assetFile.type +
+        '\\" \\/><\\/amp-video><\\/body><\\/html>"'
+      : '';
+
+    const adAmpHtml =
+      assetFile && assetFile.type.includes('video')
+        ? adHtml + ampVideoRuntime + videoHtml
+        : adHtml + imageHtml;
+
+    const storyAMPHTML = `<!DOCTYPE html>
+    <html amp="🤠-invalid" lang="en">
+      <head>
+        <meta charset="utf-8" />
+        <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+        <script async src="https://cdn.ampproject.org/v0.js"></script>
+        <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+        <script async custom-element="amp-story-auto-ads" src="https://cdn.ampproject.org/v0/amp-story-auto-ads-0.1.js"></script>
+        <title>AMP Story</title>
+        <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1" />
+        <link rel="canonical" href="/" />
+      </head>
+      <body>
+        <amp-story standalone>  
+          <amp-story-auto-ads id="i-amphtml-demo-1" development>
+            <script type="application/json">
+              {
+                "ad-attributes": {
+                  "type": "fake",
+                  "srcdoc": ${adAmpHtml},
+                  "a4a-conversion": true
+                }
+              }
+            </script>
+          </amp-story-auto-ads>
+          <amp-story-page class="i-amphtml-story-ad-cover" id="cover">
+            <amp-story-grid-layer template="fill">
+              <amp-img
+                src="https://amp.dev/static/samples/img/story_dog2.jpg"
+                width="720"
+                height="1280"
+                layout="responsive"
+              >
+              </amp-img>
+            </amp-story-grid-layer>
+            <amp-story-grid-layer template="vertical">
+              <h1>Hello World</h1>
+              <p>This is an AMP Story.</p>
+            </amp-story-grid-layer>
+          </amp-story-page>
+    
+          <amp-story-page id="page-2">
+            <amp-story-grid-layer template="fill">
+              <amp-video
```

Comment:
Can we remove the video and maybe use an image instead? Video might increase latency of preview in general.

Reply:
Will address this in a separate PR

Reply:
Created a new personalized story in changeStory branch